index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="hero">
+            <nav class="navbar">
+                <h2 class="logo">Cobby Elson</h2>
+                <button type="button" class="hamburger-btn">
+                    <span class="toggler-icon top-bar"></span>
+                    <span class="toggler-icon middle-bar"></span>
+                    <span class="toggler-icon bottom-bar"></span>
+                </button>
+            </nav>
+            <div class="about-section">
+                <div class="container-for-about-section">
+                    <div class="about-section-item">
+                        <p class="intro  ">I’m Cobby. Glad to see you!</p>
+                    </div>
+                    <div class="about-section-item">
+                        <p class="profile">I’m a software developer! I can help you build a product , 
+                            feature or website Look through some of my work and experience! 
+                            If you like what you see and have a project you need coded, 
+                            don’t hesitate to contact me.
+                        </p>
+                    </div>
+                    <div class="about-section-item">
+                        <a href="http://" class="lets-connect">LET’S CONNECT</a>
+                        <div class="social-media-icons">
+                            <img src="./assets//twitter.png" alt="twitter">
+                            <img src="./assets//Linkedin icon.png" alt="linkedin">
+                            <img src="./assets/socialIcon.png" alt="social">
+                            <img src="./assets/git.png" alt="git">
+                            <img src="./assets/twohandicon.png" alt="my pub">
+                        </div> 
+
+                    </div>
+                                          
+                </div>
+            </div>
```

Comment:
Kindly implement these changes for your `headline` section 👍 

1. The container of this section should be in a `section` tag as it represents a generic standalone section of the document, you can check this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more details
2. You did well adding some background images, kindly include all background images as your design lacks some of them 😄 
3. The heading on line 27 should be in a `h1` tag for search engine optimization about a specific topic. you can check this [link](https://www.reliablesoft.net/h1-tag/) for more details
4. The list of icons should be in  a `ul li` tag like this
   > 
         <ul>
                   <li> <img src="./assets//twitter.png" alt="twitter"></li>
                             .
                             .
                             .
           </ul>

5. Kindly use the right font-color for the paragraph `i'm a software developer which is color `#344563`, 
6. For all your texts in this section, please use the right font properties as given in the figma design so as to have a better user design and experience 😄 
7. Your social media icon doesn't seem to be properly aligned in a straight line as some has a higher width than others, could you please correct that 😃 

Figma design :
![image](https://github.com/CobbyElsonfx/Portfolio-Website/assets/20537712/50277666-0978-45d8-a563-ca7e2843af17)

Design given :
![image](https://github.com/CobbyElsonfx/Portfolio-Website/assets/20537712/8fd2ceeb-07b8-4b1e-80b5-b51526a91915)